packages/localizations/src/de-DE.ts

```diff
@@ -318,9 +318,10 @@ export const deDE: LocalizationResource = {
       title: 'Keine verfügbaren Methoden',
     },
     passkey: {
-      blockButton__passkey: undefined,
-      subtitle: undefined,
-      title: undefined,
+      blockButton__passkey: 'Benutzen Sie Ihren Passkey',
+      subtitle:
+        'Die Verwendung Ihres Passkeys bestätigt Ihre Identität. Ihr Gerät kann nach Ihrem Fingerabdruck, Gesicht oder Bildschirmsperre fragen.',
+      title: 'Benutzen Sie Ihren Passkey',
```

Comment:
```suggestion
      title: 'Verwenden Sie Ihren Passkey',
```

Let's keep things consistent